### PR TITLE
Implement PolicyAnalizer, ReportService and Python Evaluator

### PR DESCRIPTION
**Pull request by stefansjfw**

### CODE REVIEW

File: magpie-core/src/main/java/io/openraven/magpie/core/cspm/services/PolicyAnalyzerService.java [hunk 1]
```diff
@@ -21,4 +22,6 @@
    * @throws IOException
    */
   List<Violation> analyze(List<PolicyContext> policies) throws IOException;
+
+  String evaluate(Rule rule, Object resultSet) throws Exception;
```
Comment (stefansjfw) [inline magpie-core/src/main/java/io/openraven/magpie/core/cspm/services/PolicyAnalyzerService.java:26]: Doc needed

File: magpie-core/src/main/java/io/openraven/magpie/core/cspm/services/PolicyAnalyzerServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      throw new ConfigException(String.format("Config file does not contain %s configuration", PersistPlugin.ID));
+    }
+
+    try {
+      final PersistConfig persistConfig = MAPPER.treeToValue(MAPPER.valueToTree(rawPersistConfig.getConfig()), PersistConfig.class);
+
+      String url = String.format("jdbc:postgresql://%s:%s/%s", persistConfig.getHostname(), persistConfig.getPort(), persistConfig.getDatabaseName());
+      jdbi = Jdbi.create(url, persistConfig.getUser(), persistConfig.getPassword())
+              .installPlugin(new PostgresPlugin())
+              .installPlugin(new SqlObjectPlugin());
```
[the remark this answers is not in the document]
Comment (stefansjfw) [reply magpie-core/src/main/java/io/openraven/magpie/core/cspm/services/PolicyAnalyzerServiceImpl.java]: No, it's copy-pasta :) Removed. Thanks!

**Review by PrzemyslawTusinski — APPROVED** [review]

**Comment by PrzemyslawTusinski** [comment]
Code looks good. Let's merge this as well :)

**Comment by stefansjfw** [comment]
> Code looks good. Let's merge this as well :)

Needs rebasing. Will merge it first thing in the morning

**Pull request merged**